src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -597,4 +599,21 @@ public function removeParameter($parameterName)
             }
         }
     }
+
+    /**
+     * Fetch an array of all return statements found within this function.
```

Comment:
Yes, if you reflect on the closure you'll only see the return statement *inside* the closure